update_prevalence.py

```diff
@@ -357,14 +357,19 @@ def partial_vaccination_total(self):
             return reduce(lambda x, key: x + self.vaccines_by_type[key].partial_vaccinations, self.vaccines_by_type, 0)
         return self.vaccines_total.partial_vaccinations
 
-    # Compute the estimated risk ratio for an unvaccinated person vs an average person.
-    # Average prevalence = sum(prevalence in group * proportion of population in group)
-    #                    = sum(vaccine_mult * unvaccinated_prev * proportion of population)
-    # Unvaccinated risk / Average risk = sum(vaccine_mult * proportion of pop)
-    #                                  = population / sum(vaccine_mult * proportion of pop)
-    # Where the sum is taken over all vaccine types and status (incl no vaccine)
+    # Compute the estimated risk ratio for an unvaccinated person vs an average person,
+    # so that we can convert average risk (which we know from prevalence data) to
+    # unvaccinated risk if we have sufficiently detailed vaccination information.
+    #
+    # risk_sum / population will be the average vaccine multiplier across the
```

Comment:
I'm having a hard time following this new comment logic.

Could we define risk_sum before using it?

risk_sum = sum(vaccine multiplier * people with that multiplier) + 1 * unvaccinated people
risk_sum / population = avg_vaccine_multiplier
Unvaccinated people have a vaccine multiplier of 1, therefore

unvaccinated_relative_risk = 1 / avg_vaccine_multiplier